emcc.py

```diff
@@ -1146,6 +1178,7 @@ def check(input_file):
 
     if shared.Settings.STRICT:
       shared.Settings.STRICT_JS = 1
+      shared.Settings.LLD_REPORT_UNDEFINED = 1
```

Comment:
why enable this in strict? I'm not happy with strict compiles being slower.

if we have an error, can we show "link with -s LLD_REPORT_UNDEFINED to get more detailed error reporting"?

Reply:
Good idea.  Done.